docs/src/contributing_local_development.md

```diff
@@ -1,16 +1,26 @@
-### Ubuntu
-
-
-#### Build requirements:
+### Install build requirements
+#### Ubuntu
 ```
-sudo apt install git cargo libssl-dev pkg-config libpq-dev yarn curl gnupg2 git
+sudo apt install git cargo libssl-dev pkg-config libpq-dev yarn curl gnupg2
```

Comment:
Assuming this extra `git` was left in here by accident, but lmk otherwise.

Reply:
Thanks, good catch!